src/template-pages/content-template-with-search-widget.html

```diff
@@ -0,0 +1,617 @@
+<!DOCTYPE html>
```

Comment:
This is not a page template. Should be delete. 
Please add the search widget in components documentation.

Reply:
Deleted